### PR TITLE
[12.0][IMP] helpdesk_mgmt_ticket_reminder

### PR DESCRIPTION
**Pull request by lmiguens-solvos**
Add new flag to include ticket in reminder or not

### CODE REVIEW

File: helpdesk_mgmt_ticket_reminder/views/helpdesk_ticket_views.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="ticket_view_form" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.view.form (remainder_active)</field>
```
Comment (dalonsod) [inline helpdesk_mgmt_ticket_reminder/views/helpdesk_ticket_views.xml]: Typo:
```suggestion
        <field name="name">helpdesk.ticket.view.form (reminder_active)</field>
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Locally tested, reminder is only sent to reminder-set tickets 👍 
See comment, and also update `i18n` files

File: helpdesk_mgmt_ticket_reminder/views/helpdesk_ticket_views.xml (hunk 1 above)
Comment (lmiguens-solvos) [reply helpdesk_mgmt_ticket_reminder/views/helpdesk_ticket_views.xml]: Done

**Review by dalonsod — APPROVED** [review]
LGTM
Before merging please squash commits

**Pull request merged**